crates/solvers/src/tests/naive/without_pool.rs

```diff
@@ -5,7 +5,11 @@ use {crate::tests, serde_json::json};
 
 #[tokio::test]
 async fn test() {
-    let engine = tests::SolverEngine::new("naive", tests::Config::None).await;
```

Comment:
We can probably get rid of `None` here now since configuration files are required for all solvers right?

Reply:
Yes.